relengapi/blueprints/slaveloan/bugzilla.py

```diff
@@ -0,0 +1,142 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# XXX Much of this functionality should probably be its own
+#     Separate relengapi blueprint
```

Comment:
BMO functionality, imo, should not be directly part of base since its pretty specific, and we explicitly don't want base being a swiss army knife. Though it should be an SUV, where it can do a lot of things, but you need to provide the pieces to make it do them.  (to use a bad analogy)

Either way, that doesn't block slaveloan hardcoding it as its own thing for now.